src/transport/ipc-transport.js

```diff
@@ -53,7 +53,7 @@ IpcTransport.prototype.submitRpcRequest = function (rpcJso, errorCallback) {
     });
   } catch (error) {
     if (error.code === "EPIPE") error.retryable = true;
-    errorCallback(error);
+    setTimeout(function() { errorCallback(error); });
```

Comment:
Yeah, `setTimeout` with no duration is the same as `0` duration which effectively means that the timeout is elligible for execution immediately, which means the next time timeouts are executed, this one will be executed.